src/functionalTest/java/uk/gov/hmcts/reform/cdapi/AuthorizationFunctionalTest.java

```diff
@@ -84,6 +84,7 @@ public static String generateRandomEmail() {
 
     @AfterAll
     public static void destroy() {
+        log.info("Deleting user from sidam");
```

Comment:
Is it worth us including the LoggingComponentName?

Reply:
@AdilOoze its is test case side code , production logs will not have this